awx/ui/src/screens/Job/JobOutput/JobOutput.js

```diff
@@ -99,6 +99,7 @@ function JobOutput({ job, eventRelatedSearchableKeys, eventSearchableKeys }) {
   const scrollHeight = useRef(0);
   const history = useHistory();
   const eventByUuidRequests = useRef([]);
+  const eventsProcessedDelay = useRef(250);
```

Comment:
why not just use a regular `const` or `let` here instead of a useRef hook?

Reply:
Since we edit this timeout in the `pollForEventsProcessed`, we need the updated value to persist in a ref